asset-collection-example/src/main/kotlin/furhatos/app/assetcollectionexample/init.kt

```diff
@@ -1,27 +1,30 @@
 package furhatos.app.assetcollectionexample
 
-import furhat.libraries.standard.UtilsLib
-import furhatos.app.assetcollectionexample.flow.Idle
-import furhatos.app.assetcollectionexample.settings.distanceToEngage
-import furhatos.app.assetcollectionexample.settings.linkGoogleSheet
-import furhatos.app.assetcollectionexample.settings.maxNumberOfUsers
+import furhatos.app.assetcollectionexample.flow.Menu
 import furhatos.flow.kotlin.furhat
 import furhatos.flow.kotlin.state
 import furhatos.flow.kotlin.users
-import furhatos.flow.kotlin.voice.Voice
+import furhatos.util.CommonUtils
+
+val logger = CommonUtils.getLogger("AssetCollectionSkill")
 
 val Init = state {
 
     init {
         users.setSimpleEngagementPolicy(distanceToEngage, maxNumberOfUsers)
-        furhat.voice = Voice("Matthew")
+        furhat.voice = englishVoice
+        furhat.mask = "Adult"
+        furhat.character = "Alex"
 
-        /** From Asset Collection
-         * Point to the google sheet used as resource for fetching utterances and logging responses
-         * Link for the browser: https://docs.google.com/spreadsheets/d/${linkGoogleSheet}/
-         **/
-        UtilsLib.GoogleSheets.updateDefaultSheetLink(linkGoogleSheet)
+        if (users.count >= 1) {
+            furhat.attend(users.random)
+        }
+        //TODO : include official link !
+        logger.info("Official StandardLibraryCollection documentation available here : " +
+                "https://furhat-files.s3.eu-west-1.amazonaws.com/standardlibrary-docs/1.2.0-review-SNAP/-standard-library-collection/furhat.libraries.standard.utils/-google-sheets-integration/index.html")
+        furhat.say("Hello ! This skill is a demonstration of what functions are available in the Standard Library Collection and how they can be used.")
```

Comment:
In the docs we refer to it as: "the Asset Collection" that is a part of "Furhat Library". So I would like to phrase it like this instead:
"This skill is a demonstration of what functions are available in the Asset Collection Library, that is available as a part of the Furhat Library, and how these functions can be used. Read more about the Furhat Library in the docs page. "

Reply:
Actually. Perhaps it would be smart with an "about this skill" button that could give some further explanation. And we can keep this intro short and sweet! 
"This skill is a demonstration of what functions are available in the Asset Collection Library and how these functions can be used. "